Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.17 as builder
+FROM golang:1.18 as builder
```

```suggestion
FROM golang:1.17 as builder
```
> `go mod tidy: go.mod file indicates go 1.18, but maximum supported version is 1.17` our integration tests need updating to support golang 1.18, we could cap it at 1.17 until this support is added.

Keep it at 1.17 for now.